cf/field.py

```diff
@@ -13454,42 +13455,43 @@ def halo(
+            depth: `int` or `dict`
                 Specify the size of the halo for each axis.
 
-                If *size* is a non-negative `int` then this is the halo
-                size that is applied to all of the axes defined by the
-                *axes* parameter.
+                If *depth* is a non-negative `int` then this is the
+                halo size that is applied to all of the axes defined
+                by the *axes* parameter.
 
                 Alternatively, halo sizes may be assigned to axes
                 individually by providing a `dict` for which a key
-                specifies an axis (by passing the axis description to a
-                call of the field construct's `domain_axis` method. For
-                example, for a value of ``'X'``, the domain axis construct
-                returned by ``f.domain_axis('X')``) with a corresponding
-                value of the halo size for that axis. Axes not specified
-                by the dictionary are not expanded, and the *axes*
-                parameter must not also be set.
+                specifies an axis (by passing the axis description to
+                a call of the field construct's `domain_axis`
+                method. For example, for a value of ``'X'``, the
+                domain axis construct returned by
+                ``f.domain_axis('X')``) with a corresponding value of
+                the halo size for that axis. Axes not specified by the
+                dictionary are not expanded, and the *axes* parameter
+                must not also be set.
 
                 *Parameter example:*
                   Specify a halo size of 1 for all otherwise selected
-                  axes: ``size=1``
+                  axes: ``1``
 
                 *Parameter example:*
-                  Specify a halo size of zero ``size=0``. This results in
+                  Specify a halo size of zero: ``0``. This results in
                   no change to the data shape.
 
                 *Parameter example:*
-                  For data with three dimensions, specify a halo size of 3
-                  for the first dimension and 1 for the second dimension:
-                  ``size={0: 3, 1: 1}``. This is equivalent to ``size={0:
-                  3, 1: 1, 2: 0}``
+                  For data with three dimensions, specify a halo size
+                  of 3 for the first dimension and 1 for the second
+                  dimension: ``{0: 3, 1: 1}``. This is equivalent to
+                  ``{0: 3, 1: 1, 2: 0}``
 
                 *Parameter example:*
                   Specify a halo size of 2 for the "longitude" and
-                  "latitude" axes: ``size=2, axes=['latutude',
-                  'longitude']``, or equivalently ``size={'latutude': 2,
-                  'longitude': 2}``.
+                  "latitude" axes: ``depth=2, axes=['latutude',
+                  'longitude']``, or equivalently ``depth={'latutude':
```

Comment:
Not a typo introduced in this PR, but it would be appropriate to mop it up here:

```suggestion
                  'longitude']``, or equivalently ``depth={'latitude':
```